scripts/plot_timeline.py

```diff
@@ -91,48 +137,86 @@ def get_pipeline_creation_times(create_pipeline_events):
     ys = []
 
     for event in create_pipeline_events:
-        timepoint, _hash, creation_time_nanos = event[0], event[1], int(event[2])
-        elapsed_seconds = timepoint / nanos_per_second
+        elapsed_seconds = event.timestamp / nanos_per_second
         xs.append(elapsed_seconds)
-        ys.append(creation_time_nanos / nanos_per_milli)
+        ys.append(event.duration / nanos_per_milli)
 
     return xs, ys
 
+
 def plot_frames_per_second(ax, events_by_type):
     ax.set_xlabel('Time Since Start [s]')
     ax.set_ylabel('Frames Per Second')
 
     if 'frame_present' not in events_by_type:
         return
 
-    fps_x, fps_y, states = get_frames_per_second(events_by_type['frame_present'])
+    fps_x, fps_y, states = get_frames_per_second(
+        events_by_type['frame_present'])
     for state, xs, ys in split_by_state(fps_x, fps_y, states):
-        ax.plot(xs, ys, color=plt.cm.tab10(state), label=f'FPS in state {state}')
+        ax.plot(xs, ys, color=plt.cm.tab10(state),
+                label=f'FPS in state {state}')
 
     ax.legend(loc='upper right')
     ax.grid()
 
+
 def plot_pipeline_creations(ax, events_by_type):
     ax.set_ylabel('Creation Time [ms]')
     max_creation_time_millis = 0
 
     if 'create_graphics_pipelines' in events_by_type:
-        xs, ys = get_pipeline_creation_times(events_by_type['create_graphics_pipelines'])
-        ax.scatter(xs, ys, s=3, color=plt.cm.tab10(8), label='Create Graphics Pipelines')
+        xs, ys = get_pipeline_creation_times(
+            events_by_type['create_graphics_pipelines'])
+        ax.scatter(xs, ys, s=3, color=plt.cm.tab10(
+            8), label='Create Graphics Pipelines')
```

Comment:
I think we should keep these unrelated formatting changes separate. Either undo them here and elsewhere or open a PR to format the code you are going to modify first.

Reply:
Also elsewhere in this file.

Reply:
Sure!